docs/using/administering/debugging.rst

```diff
@@ -33,6 +35,46 @@ It is possible to see the available unit names, and then view the logs with ``jo
    [root@node1]# journalctl -u flocker-container-agent
    [root@node1]# journalctl -u flocker-control
 
+Bug Reporting
+-------------
+
+When reporting issues with Flocker please include:
+
+* Operating System and version,
```

Comment:
Yep, we only use title case in our headers - in this context it should be Operating system